rails-ubuntu-simple/README.md

```diff
@@ -0,0 +1,239 @@
+---
+layout: guides
+title: Running a load balanced Ruby on Rails app with Weave, and Docker
+description: "Running a load balanced Ruby on Rails app with Weave, and Docker"
+keywords: "weave, docker, containers, networking, software defined networking, ubuntu, ruby, rails, load blancing, microservices, dns, pg, postgres, postgresql"
+markdown: kramdown
+highlighter: pygments
+---
+
+
+## What you will build ##
+
+Weave allows you to focus on developing your application, rather than
+your infrastructure.
+
+In this example we will set up a simple, containerized deployment of a
+Ruby on Rails application, backed by a standard MySQL database.
+
+We will use
+[WeaveDNS](https://github.com/weaveworks/weave/tree/master/weavedns#readme)
+for service discovery between the Rails application, and its database,
+without changing any code.
+
+## What you will use ##
+
+* [Weave](http://weave.works)
+* [Docker](http://docker.com)
+* [Rails](http://rubyonrails.org)
+* [PostgreSQL](http://www.postgresql.org)
+* [Ubuntu](http://ubuntu.com)
+
+## What you will need to complete this guide ##
+
+This getting started guide is self contained. You will use Weave, Docker, Rails, and Ubuntu. We make use of VirtualBox and Vagrant to allow you to run this entire getting started guide on your personal system.
+
+* 20 minutes
+* [Git](http://git-scm.com/downloads)
+* [VirtualBox > 4.3.20](https://www.virtualbox.org/wiki/Downloads)
+* [Vagrant > 1.6](https://docs.vagrantup.com/v2/installation/index.html)
+
+## Setting up our hosts ##
+
+All of the code for this example is available on github, and you first clone the getting started repository.
+
+```bash
+$ git clone http://github.com/fintanr/weave-gs
+```
+
+You will use Vagrant to setup and configure three Ubuntu hosts and
+install Docker. We make use of Vagrant's functionality to download the
+base docker images we will be using, and we then install Weave. If you
+would like to work through the installation steps please review our
+[getting started
+guide](https://github.com/fintanr/weave-gs/blob/master/ubuntu-simple/README.md)
+for a more manual example.
+
+```bash
+$ cd weave-gs/rails-ubuntu-simple
+$ vagrant up
+```
+
+Vagrant will pull down and configure an Ubuntu image, this may take a
+few minutes depending on the speed of your network connection. For
+more details on Vagrant please refer to the [Vagrant
+documentation](http://vagrantup.com).
+
+You may be prompted for a password when `/etc/hosts` is being updated
+during the Vagrant setup, please just hit return at this point.
+
+Once the setup of the host is complete you can check its status with
+
+```bash
+$ vagrant status
+```
+
+The IP address we use for this is
+
+```bash
+172.17.8.101 	weave-rails-01
+```
+
+Our Vagrantfile also configures weave-rails-01 to pass traffic from port 3000 to localhost port 3000.
```

Comment:
This seems to need some explanation, e.g. "which we will use later when we connect to the service"